client/landing/stepper/declarative-flow/internals/steps-repository/sensei-launch/launch-completion-tasks.ts

```diff
@@ -0,0 +1,34 @@
+import wp from 'calypso/lib/wp';
+
+const PERSISTENT_DATA_DELAY = 1200;
+
+function waitMs( ms: number ) {
```

Comment:
Could we import the same `wait` used in the other file? So we avoid another function for the same purpose.

Reply:
Actually, to follow the same standard, maybe we could also call this `wait` there? WDYT?